src/MailThief.php

```diff
@@ -101,7 +101,7 @@ public function later($delay, $view, array $data, $callback, $queue = null)
 
     public function hasMessageFor($email)
     {
-        return $this->messages->contains(function ($i, $message) use ($email) {
+        return $this->messages->contains(function ($i, Message $message) use ($email) {
```

Comment:
For example, in PhpStorm, without the typehint:
![image](https://cloud.githubusercontent.com/assets/2274910/16897850/e1fc8c62-4b95-11e6-9230-c2dad725e06d.png)

With the typehint you'd be able to navigate to the `hasRecipient` method. Do you think it's worth? :)